src/components/ShowHide/ShowHide.module.css

```diff
@@ -12,39 +12,30 @@
   display: none;
 
   /**
-     * 1) Button that toggles open and closes a panel
-     */
+   * 1) Button that toggles open and closes a panel
+   */
   .show-hide.eds-is-active & {
     margin-top: var(--eds-size-1);
     display: block;
   }
 }
 
 /**
-   * Button icon within show hide component
-   * 1) Add transition for icon flip when activated
-   */
-.show-hide__btn .button__icon {
```

Comment:
The `show-hide__btn`, `button__icon`, and `show-hide--sm` classes don't actually exist in the component.